changelogs/unreleased/fix-exception-typo.yml

```
@@ -0,0 +1,5 @@
+description: Fix jwt config error message to use the correct attribute and provide more context
+change-type: patch
+destination-branches: [master, iso6]
+sections:
+  feature: "{{description}}"
```

```suggestion
  bugfix: "{{description}}"
```
Right? Otherwise at least `minor-improvement` rather than `feature`.

---

indeed